lib/components/BuilderForm.jsx

```diff
@@ -40,54 +40,56 @@ export const BuilderForm = (props) => {
     setMaxExitFeeMantissa,
     setMaxTimelockDuration,
     setTicketCreditLimitMantissa,
-    setExternalERC20Awards,
+    setExternalERC20Awards
   } = stateSetters
 
-  const handleTickerChange = (newToken) => {
+  const handleTickerChange = newToken => {
     setCToken(newToken)
   }
-  const handleRngServiceChange = (e) => {
+  const handleRngServiceChange = e => {
     setRngService(e.target.value)
   }
 
-  return <>
-    <form
-      onSubmit={handleSubmit}
-    >
-      <div
-        className='font-bold mb-8 py-2 text-lg sm:text-xl lg:text-2xl text-default'
-      >
-        Prize Pool Parameters:
-      </div>
-
-      <label
-        htmlFor={'token'}
-        className='mt-0 trans text-purple-300 hover:text-white'
-      >
-        Yield service token to use:
-      </label>
-      <TokenDropdown
-        onChange={handleTickerChange}
-      />
-
-      <RadioInputGroup
-        label={<>RNG Service: <span className='text-default italic'>(Random number generator)</span></>}
-        name='_rngService'
-        onChange={handleRngServiceChange}
-        value={rngService}
-        radios={[
-          {
-            value: 'blockhash',
-            label: 'Blockhash'
-          },
-          {
-            value: 'chainlink',
-            label: 'Chainlink'
-          },
-        ]}
-      />
-
-      {/* <TextInputGroup
+  return (
```

Comment:
We should be able to. I'll go through your comments & add rules for everything.

Reply:
Ah oh https://github.com/prettier/prettier/issues/2724

Reply:
hmm, that's annoying. I wonder how hard it would be to patch prettier and submit a PR.

That or use something else like standard, and just have it behave similar to prettier.

Reply:
Oh just saw the maintainers note about not accepting a PR for this. Well we can fork it and just run our own version of it.

Reply:
It looks like the parenthesis are just the default accepted way of formatting multi-line jsx. Standard is super opinionated and doesn't encourage customization. I'll look a little more tomorrow. Tbh I kinda like it now that I've read some examples of why it's the standard. Keeping the HTML tags indented properly is pretty.